src/main/java/ua/hillel/treshchun/lessons/lesson12HW15WorkingWithFiles/FileLoggerConfigurationLoader.java

```diff
@@ -0,0 +1,100 @@
+package ua.hillel.treshchun.lessons.lesson12HW15WorkingWithFiles;
+
+import java.io.*;
+
+public class FileLoggerConfigurationLoader implements LoggerConfigurationLoader {
+
+
+    public FileLoggerConfiguration load(String fileName) throws FileConfigurationException {
```

Comment:
I believe complexity of load(String fileName) method can be simplified and code can be reduced in number of lines and logic